api/swagger.yaml

```diff
@@ -18,6 +18,9 @@ definitions:
       id:
         type: 'integer'
         format: 'uint32'
+      requestTime: 
+        type: 'string'
+        format: 'date-time'
       time:
         type: 'string'
         format: 'date-time'
```

Comment:
@aduminuc

Reply:
time is the time when the request was received by APIClarity, requestTime is the time inserted by kong or envoy or other GWs. The two do not necessarily matches. Especially if you use traces instead of real traffic.

Reply:
The question why we need both? why we should not use request time always and if missing (from traces) we can use the time it was received be APIClarity.

Reply:
If you use a mix of them you may end up with misordering of events. If we are ingesting events from traces the API clarity receive time can be completely different from the trace time.

Reply:
So the two will appear in the UI?

Reply:
if the two appear in the UI, maybe I missed it, but sort and filter is also needed, no?

Reply:
I was thinking of adding it to the event detail, not in the event table. But we can add in the table as well. Good point.

Reply:
On the other hand maybe we shouldn't add to the table. Don't want it to interfere with time filter stuff.

Reply:
In case we add it, we should make it clear what we are filtering on.